tests/h/models/group_test.py

```diff
@@ -238,259 +238,6 @@ def test_non_public_group():
-    ):
-        group.authority = "weewhack.com"
-
-        assert authz_policy.permits(
-            group, ["flip", "client_authority:weewhack.com"], Permission.Group.ADMIN
-        )
-
-    def test_auth_client_without_matching_authority_does_not_have_admin_permission(
-        self, group, authz_policy
-    ):
-        group.authority = "weewhack.com"
-
-        assert not authz_policy.permits(
-            group, ["flip", "client_authority:2weewhack.com"], Permission.Group.ADMIN
-        )
-
-    def test_creator_has_upsert_permissions(self, group, authz_policy):
-        assert authz_policy.permits(
-            group, "acct:luke@example.com", Permission.Group.UPSERT
-        )
-
-    def test_admin_allowed_only_for_authority_when_no_creator(
-        self, group, authz_policy
-    ):
-        group.creator = None
-
-        principals = authz_policy.principals_allowed_by_permission(
-            group, Permission.Group.ADMIN
-        )
-
-        assert "client_authority:example.com" in principals
-        assert authz_policy.permits(
-            group, ["flip", "client_authority:example.com"], Permission.Group.ADMIN
-        )
-
-    def test_no_moderate_permission_when_no_creator(self, group, authz_policy):
-        group.creator = None
-
-        principals = authz_policy.principals_allowed_by_permission(
-            group, Permission.Group.MODERATE
-        )
-        assert len(principals) == 0
-
-    def test_no_upsert_permission_when_no_creator(self, group, authz_policy):
-        group.creator = None
-
-        principals = authz_policy.principals_allowed_by_permission(
-            group, Permission.Group.UPSERT
-        )
-        assert len(principals) == 0
-
-    def test_staff_user_has_admin_permission_on_any_group(self, group, authz_policy):
-        principals = authz_policy.principals_allowed_by_permission(
-            group, Permission.Group.ADMIN
-        )
-
-        assert role.Staff in principals
-        assert authz_policy.permits(
-            group, ["whatever", "group:__staff__"], Permission.Group.ADMIN
-        )
-
-    def test_admin_user_has_admin_permission_on_any_group(self, group, authz_policy):
-        principals = authz_policy.principals_allowed_by_permission(
-            group, Permission.Group.ADMIN
-        )
-
-        assert role.Admin in principals
-        assert authz_policy.permits(
-            group, ["whatever", "group:__admin__"], Permission.Group.ADMIN
-        )
-
-    def test_fallback_is_deny_all(self, group, authz_policy):
-        assert not authz_policy.permits(group, [security.Everyone], "foobar")
-
-    @pytest.fixture
-    def authz_policy(self):
-        return ACLAuthorizationPolicy()
-
-    @pytest.fixture
-    def group(self):
-        creator = models.User(username="luke", authority="example.com")
-        group = models.Group(
-            name="test-group", authority="example.com", creator=creator
-        )
-        group.pubid = "test-group"
```

Comment:
All of these magic values have been replaced with interpolation instead, which allows this to be replaced with some simple factories.